python/scope.py

```diff
@@ -767,7 +767,7 @@ def type_check_expr(self, expr):
 			func_type = self.type_check_expr(function)
 			if func_type is None:
 				return None
-			if not isinstance(func_type, tuple):
+			if not isinstance(func_type, tuple) and not isinstance(func_type, NModule):
```

Comment:
??? Why would you allow calling modules? If I'm not mistaken, NModules are (they extend) dictionaries

Reply:
That was a mistake, ill remove it in the next commit